src/server/page.tsx

```diff
@@ -60,6 +60,7 @@ const template = (
     Sentry.init(${JSON.stringify(sentryConfig())})
   </script>
   <script key="segment-snippet" nonce="${cspNonce}">${segmentSnippet}</script>
+  <script defer src="https://adtr.io/jsTag?ap=1412531808"></script>
```

Comment:
this (and any other domains they might use) must be placed in the CSP headers

Reply:
Good catch! 💯